projects/ardin_mb/ardin_mb.cc

```diff
@@ -35,6 +34,26 @@ using namespace BoBRobotics;
 
 int main(int argc, char *argv[])
 {
+    // Default parameters"
+    std::string worldFilename = "";
+    std::string routeFilename = "";
+    std::string logFilename = "";
+    float heightMetres = 0.01f;
+    std::vector<float> minBound;
+    std::vector<float> maxBound;
+    std::vector<float> clearColour{0.0f, 1.0f, 1.0f, 1.0f};
```

Comment:
I'm guessing ``std::array<...>`` wouldn't work instead here?

Reply:
Sadly not - I would agree that that would be the correct container for this use-case but [only std::vector is supported](https://github.com/BrainsOnBoard/bob_robotics/blob/master/third_party/CLI11.hpp#L2646-L2688)